packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1255,6 +1259,16 @@ function build(
     // we do not use vreffedObjectRegistry for root objects
   }
 
+  /**
+   * @returns { Promise<void> }
+   */
+  async function stopVat() {
+    assert(didStartVat);
+    assert(!didStopVat);
+    didStopVat = true;
```

Comment:
This assignment doesn't seem entirely no-op'y, since it will cause `notify` and `deliver` to throw.

Reply:
yeah, I'm ok with that, it's less "no-op" and more "don't do nearly enough to qualify for the `stopVat` name but still let's establish exactly when and how this should be called and we'll fill in the rest of the work later" :)